base/loading.jl

```diff
@@ -2509,6 +2510,17 @@ function create_expr_cache(pkg::PkgId, input::String, output::String, output_o::
     return io
 end
 
+const precompilation_stack = Vector{String}()
```

Comment:
This needs to be `Vector{PkgId}` but there's currently no easy way to print that as machine readable.

Reply:
You can create it manually via something like `"Base.PkgId($(reor(pkg.uuid)) , $(repr(pkg.name)))" `

Reply:
The tricky part is that I want to pass in a `Vector{PkgId}`.

I can make a helper as [changing the default show for `PkgId`](https://github.com/JuliaLang/julia/pull/52795) seems like it would break a lot of places where `"$pkg something"` is used

Reply:
> seems like it would break a lot of places where "$pkg something" is used

Shouldn't that be a countable number of places in Base that can be fixed up?